app/femr/data/models/mysql/RankedPatientMatch.java

```diff
@@ -0,0 +1,38 @@
+package femr.data.models.mysql;
+import femr.data.models.core.IRankedPatientMatch;
+import io.ebean.annotation.Sql;
+import javax.persistence.*;
+
+@Entity
+@Sql
+public class RankedPatientMatch implements IRankedPatientMatch {
```

Comment:
might want to add a test class for this later

Reply:
I added a Mock class that implements IRankedPatientMatch to be used in place of this class in tests.